modules/KalturaSupport/resources/mw.KEntryLoader.js

```diff
@@ -36,9 +36,11 @@ mw.KEntryLoader.prototype = {
 		}
 		// Local method to fill the cache key and run the associated callback
 		var fillCacheAndRunCallback = function( namedData ){
-			_this.playerLoaderCache[ _this.getCacheKey( kProperties ) ] = namedData;
+			if ( !mw.getConfig("DisableEntryCache") ) {
```

Comment:
@itaykinnrot - maybe change the Flashvar name to "EmbedPlayer.DisableEntryCache", add a description to https://github.com/kaltura/mwEmbed/blob/2.36-Oct-20/docs/api_uivars.php#L809 and link the test page "PlaylistEvents.qunit.html" as an example?